libbeat/docs/processors-using.asciidoc

```diff
@@ -531,6 +532,52 @@ section.
 NOTE: If you define an empty list of fields under `include_fields`, then only
 the required fields, `@timestamp` and `type`, are exported.
 
+[[rename-fields]]
+=== Rename fields from events
+
+The `rename` processor specifies a list of fields that should be renamed. The
+list of fields to be renamed are listed under `fields`. Each entry contains a
```

Comment:
"list of fields to be renamed are listed under `fields`" seems a little redundant.

Reply:
changed